src/templates/template-text-classification/README.md

```diff
@@ -31,7 +31,7 @@ python -m torch.distributed.launch \
   --master_addr #:::= it.master_addr :::# \
   --master_port #:::= it.master_port :::# \
   --use_env main.py \
-  --backend nccl
+  --backend #:::= backend :::#
```

Comment:
Can you try `#:::= it.backend :::#`, I think there should be a typo on `#:::= nproc_per_node :::#` which should be `#:::= it.nproc_per_node :::#`

@ydcjeff why we have `#:::= nproc_per_node :::#` and just above `#::: if (it.nproc_per_node) { :::#` ?

Reply:
Yea, i think it should include `it.`.